physics/gravitational_force.cpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file gravitational_force.cpp
+ * @brief
+ * https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation
+ * @details
+ *newton's law of universal gravitation
+ * @author [Ayaanlehashi11](https://github.com/ayaanlehashi11)
+ * @see
+ */
+
+#include <iostream>
+#include <cmath>
+#include <cstdint>
+
+# define G_CONSTANT 6.6743e-11
+# define EARTH_MASS 5.98e24
+# define EARTH_RADIUS 6.39e+6
```

Comment:
IMO, it's better to use these directly in the function rather than using `define`.